tasks/configure.yml

```diff
@@ -95,18 +95,18 @@
   loop: "{{ barman_check_result.results }}"
   loop_control:
     label: "{{ item.pg_server_name }}"
-  when: item.stdout_lines | select('match', '^' ~ item.pg_server_name ~ '\.WAL archive:\ FAILED') | list | length > 0
+  when: barman_run_post_config_check and item.stdout_lines | select('match', '^' ~ item.pg_server_name ~ '\.WAL archive:\ FAILED') | list | length > 0
```

Comment:
Disabling creation of slots and triggering WAL archiving should have their own variables, e.g. barman_create_slots, barman_trigger_wal_archiving, but why do these two need disabling?

Reply:
Sure, allows even more fine tuned control.

The use case would be when you want to setup rsync backup with WAL archiving using `barman-wal-archive`. That is the path we've chosen for our backups.

Reply:
In my opinion that makes it easier to perform a database failover (in a primary-standby setup) when you don't have to change anything on the barman server. One just have to make changes on the database servers and barman will unaware of any changes. Assuming SSH keys are reused between the primary and the standby.

Reply:
Failover can be handled by Barman when streaming as well btw by specifying multiple hosts and `target_session_attrs=read-write`

Reply:
TIL, thanks for sharing.